Pet Grooming Software/3 Reports.sql

```diff
@@ -0,0 +1,26 @@
+
+-- 1)We need a list of how many of each pet we currently have.
+
+select HowManyPetsPerType = count(*), p.petType
+from petGroomerData p
+group by petType
+
+-- 2)We need a list of those who have multiple pets, include how many pets they have.
+
+select p.firstName, p.lastName, count(*) as PetsOwned
+from petGroomerData p 
+group by p.firstName, p.lastName
+having count(*) > 1
+
+-- 3)We need to know our top customer - the customer that is paying us the most.
```

Comment:
I'm supposed to see who's paying the most per week?

Reply:
Not specifically per week, just the customer that is paying the most - and that total price is affected by frequency of service - if the customer gets grooming twice a week for a pet, he is paying more.
Regardless, you don't need to implement this as it would involve code that you weren't taught yet.  I just wanted to bring it up to make you aware that technically this could use improvement at a later point